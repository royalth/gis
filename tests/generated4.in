100 100
91 40
67 59
79 51
14 83
75 48
21 72
51 43
42 86
79 35
84 80
22 14
39 88
28 83
40 27
13 51
54 12
28 48
11 23
10 30
83 87
92 52
1 14
10 18
85 69
90 58
76 45
59 81
38 98
90 29
81 11
59 18
17 81
71 17
61 37
52 93
21 79
45 65
75 12
60 90
24 76
80 5
8 13
98 28
70 43
37 43
93 51
74 64
57 41
94 44
90 98
68 5
15 42
53 85
76 77
58 53
50 18
51 58
65 30
22 10
55 76
100 35
98 68
96 72
19 7
29 44
86 12
45 64
63 34
82 7
41 63
39 12
20 53
31 45
37 49
19 9
13 16
65 62
6 98
33 43
66 74
79 9
70 99
97 93
46 81
49 57
17 47
91 26
89 49
27 77
89 73
25 98
68 3
32 83
9 56
4 47
51 78
86 36
98 2
75 95
41 5
