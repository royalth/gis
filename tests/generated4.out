7
1 14 1
2 98 1
3 68 4
4 47 2
5 41 3
5 68 5
5 80 4
6 98 2
7 19 2
7 82 1
8 13 2
9 19 1
9 56 3
9 79 2
10 18 2
10 22 1
10 30 3
11 23 2
11 81 1
12 39 1
12 54 3
12 75 2
12 86 4
13 16 3
13 51 1
14 22 2
14 83 3
15 42 2
17 47 1
17 71 2
17 81 3
18 50 1
18 59 3
20 53 2
21 72 2
21 79 3
24 76 1
25 98 4
26 91 1
27 40 1
27 77 2
28 48 2
28 83 1
28 98 3
29 44 3
29 90 4
30 65 2
31 45 2
32 83 2
33 43 1
34 63 1
35 79 4
35 100 3
36 86 2
37 43 3
37 49 1
37 61 2
38 98 5
39 88 2
40 91 2
41 57 1
41 63 2
42 86 3
43 51 2
43 70 4
44 94 2
45 64 3
45 65 1
45 76 4
46 81 4
48 75 1
49 57 2
49 89 3
51 58 3
51 78 4
51 79 5
51 93 6
52 92 3
52 93 4
53 58 4
53 85 3
55 76 2
58 90 5
59 67 1
59 81 2
60 90 6
62 65 3
64 74 2
66 74 1
68 98 6
69 85 2
70 99 3
72 96 1
73 89 2
75 95 3
76 77 3
80 84 3
83 87 4
90 98 7
93 97 5
